src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixActionsSource.cs

```diff
@@ -74,21 +74,28 @@ internal sealed class QuickFixActionsSource : ISuggestedActionsSource
         public event EventHandler<EventArgs> SuggestedActionsChanged;
 
         public IEnumerable<SuggestedActionSet> GetSuggestedActions(
-            ISuggestedActionCategorySet requestedActionCategories, 
+            ISuggestedActionCategorySet requestedActionCategories,
             SnapshotSpan range,
             CancellationToken cancellationToken)
         {
             var allActions = new List<ISuggestedAction>();
 
-            if (IsOnIssueWithApplicableQuickFixes(range, out var issuesWithFixes))
```

Comment:
The GitHub diff isn't helpful here. None of the existing code has changed. It's just been moved inside the try-catch.

Same for the other method.